docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
@spion Ok, now I understand the problem. Global events subscriptions are safer to implement than handling this problem in other ways. What do you think is this acceptable to store an array with bluebird instances on (process|window)?